_includes/css/activation.scss.liquid

```diff
@@ -236,9 +288,9 @@
   The following rule styles the link to the current page.
 {%- endcomment %}
 
-{{ activation_collection_prefix }} > li:nth-child({{ activation_first_level_index }})
-{%- if activation_second_level_index %} > ul > li:nth-child({{ activation_second_level_index }})
-{%- if activation_third_level_index %} > ul > li:nth-child({{ activation_third_level_index }})
+{{ activation_collection_prefix }} > li:nth-child({{ nav_levels[1] }})
+{%- if nav_levels[2] %} > ul > li:nth-child({{ nav_levels[2] }})
+{%- if nav_levels[3] %} > ul > li:nth-child({{ nav_levels[3] }})
```

Comment:
I *think* this line also has the wrong `nav_levels[3]`? Which would explain

```
diff --color -rw -x *.js* -x *.map -I published_time -I dateModified -I <meta property= -I <link rel= -I @context _site/navigation/order/mixture/0/index.html _site-0.7.0/navigation/order/mixture/0/index.html
19c19
< .site-nav > ul:nth-of-type(8) > li > ul > li:nth-child(6) > ul > li:nth-child(5) > ul > li:nth-child(3) > a {
---
> .site-nav > ul:nth-of-type(8) > li > ul > li:nth-child(6) > ul > li:nth-child(5) > ul > li:nth-child(4) > a {
```

and

```
diff --color -rw -x *.js* -x *.map -I published_time -I dateModified -I <meta property= -I <link rel= -I @context _site/navigation/order/mixture/0.0/index.html _site-0.7.0/navigation/order/mixture/0.0/index.html
19c19
< .site-nav > ul:nth-of-type(8) > li > ul > li:nth-child(6) > ul > li:nth-child(5) > ul > li:nth-child(4) > a {
---
> .site-nav > ul:nth-of-type(8) > li > ul > li:nth-child(6) > ul > li:nth-child(5) > ul > li:nth-child(3) > a {
```

Reply:
I can't imagine any way in which the changes in this PR could make a difference to the order of the cached navigation panel.

The diff suggested that Liquid/Jekyll doesn't sort `0` and `0.0` in a particular order, and I've now verified that.